web/pageComponents/shared/portableText/components/BulletList.tsx

```diff
@@ -1,5 +1,10 @@
 import { List } from '@components'
+import styled from 'styled-components'
+
+const StyledList = styled(List)`
+  margin-bottom: var(--space-medium);
```

Comment:
I think this is the best solution given the current setup we have (in which I hate - adding margin bottom to elements instead of gaps to the parent div). However, we need to keep in mind that for use cases we could end up with double margins. Example, an ingress that applies margin-bottom, and ends with a styled list. Nothing to change here, but just to keep in mind that these can occur, then we fix case by case when reported